### ISSUE TITLE
Exception when using Stroke

### ISSUE THREAD

**Issue by TomQv** [Robmaister/SharpFont#115]
I can't get Stroke up and running.

```
_lib = new Library();
_fontFace=new Face(_lib, fontfile);
_fontFace.SetCharSize(_size, _size, 96, 96);
_stroker = new Stroker(_lib);
_stroker.Set(96, StrokerLineCap.Round, StrokerLineJoin.Bevel, 0);

_fontFace.LoadChar(text[0], LoadFlags.Render, LoadTarget.Normal);
var glyph = _fontFace.Glyph.GetGlyph();
glyph.Stroke (_stroker,  true );
FTBitmap ftbmp = _fontFace.Glyph.Bitmap;
```

Ends with this exception in glyph.Stroke .
Any ideas??

SharpFont.FreeTypeException occurred
  HResult=-2146233088
  Message=FreeType error: Invalid argument.
  Source=SharpFont
  StackTrace:
       bei SharpFont.Glyph.Stroke(Stroker stroker, Boolean destroy) in SharpFont-master\Source\SharpFont\Glyph.cs:Zeile 365.
  InnerException:

**Comment by HinTak** [comment]
It is what it says. One possibility is that you are passing a pure bitmap font for stroking, such as Apple's Color Emoji. 

You'll have to say what font you are passing in, or share that font privately, for others to have a look.

**Comment by TomQv** [comment]
I'm playing with the samples from this repository, using for example AlexBrush-Regular.ttf.
Normal text-rendering works.

**Comment by HinTak** [comment]
I have not used the C# part of sharpfont for this, but the python equivalent of what you are trying to do is :
https://github.com/HinTak/freetype-py/blob/master/examples/glyph-outline.py

And the glyph-outline-cairo example in the same directory, and those certainly work.  You might be able to learn something from studying or running the python code.

**Comment by HinTak** [comment]
Hmm, that's not surprising - having worked in both sharpfont and freetype-py, the sort of bugs I file for freetype-py tends to be enhancement/addition bugs I.e. APIs are missing, while for sharpfont, it tends to be "you have it but it does not work". I.e. API in sharpfont are more complete but often untested or have never been used. Hence my suggestion to look at the python code.

**Comment by Robmaister** [comment]
I believe this is now fixed with the merge of #125
